packages/core/src/connectors/rest.js

```diff
@@ -20,7 +20,7 @@ export function restConnector(uri = 'http://localhost:3000/') {
       });
 
       return query.type === 'exec' ? req
-        : query.type === 'arrow' ? tableFromIPC(req)
+        : query.type === 'arrow' ? decodeIPC(await (await req).arrayBuffer())
```

Comment:
It would be nice to support streams here (at some point) so the client can process data while it's still being sent. See https://jakearchibald.com/2016/streams-ftw/.